ptvsd/wrapper.py

```diff
@@ -129,6 +130,47 @@ def using_format(self, fmt):
 str_handlers = pydevd_extutil.EXTENSION_MANAGER_INSTANCE.type_to_instance.setdefault(pydevd_extapi.StrPresentationProvider, [])  # noqa
 str_handlers.insert(0, SafeReprPresentationProvider._instance)
 
+DONT_TRACE_FILES = {
+    os.path.join('ptvsd', 'attach_server.py'),
```

Comment:
One other thought - if you just put tuples here, i.e.:
```py
{
  ('ptvsd', 'attach_server.py'),
  ('ptvsd', 'daemon.py'),
  ...
}
```
then you can iterate over that list and just invoke `os.path.join(*elem)` for every element, to avoid having to repeat it over and over again.